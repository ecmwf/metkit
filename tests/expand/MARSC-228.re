retrieve,
date=20210501,time=2101/to/2400,type=ob,target="/ec/fws2/sb/work/rd/erds/i7ug/LWDA/2021050200/an/scat",
target="reference.xoB8Cx.data"