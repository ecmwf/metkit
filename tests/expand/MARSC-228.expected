RETRIEVE,
    CLASS      = OD,
    TYPE       = OB,
    STREAM     = OPER,
    EXPVER     = 0001,
    REPRES     = BU,
    OBSTYPE    = 1,
    DATE       = 20210501,
    TIME       = 2101,
    DOMAIN     = G,
    TARGET     = reference.xoB8Cx.data,
    DUPLICATES = KEEP,
    RANGE      = 179